include/bdsm/export.h

```diff
@@ -0,0 +1,40 @@
+/*****************************************************************************
```

Comment:
Probably a bit pointless to have this in an installed header, as you do not actually want to export any dsm symbols in a library that uses dsm but there just have BDSM_EXPORT be a no-op.

Reply:
BDSM_EXPORT need to be defined, indeed it could be nothing on installed headers, but I don't really see how to do it.

Reply:
Actually it is useful as we could add the proper  `__declspec(dllimport)` for Windows when DLL_EXPORT is not defined and the usage-side is not using the static library.

However I can add this after this is merged.